plugins/header-classifier/src/main/java/io/buoyant/http/classifiers/HeaderClassifierInitializer.java

```diff
@@ -0,0 +1,15 @@
+package io.buoyant.http.classifiers;
+
+import io.buoyant.linkerd.ResponseClassifierInitializer;
+
+public class HeaderClassifierInitializer extends ResponseClassifierInitializer {
+    @Override
+    public String configId() {
+        return "io.buoyant.headerClassifier";
```

Comment:
My plan is to write a more tutorial-y plugin guide on the linkerd site that explains what each of these things are and how they relate to one another.  Basically this example is reference material for the tutorial (which I have yet to write).